src/ui/buttons/buttons.jsx

```diff
@@ -242,6 +285,14 @@ export function Buttons(props: ButtonsProps): ElementNode {
     return i;
   };
 
+  const calculatedMsgPosition = calculateMsgPosition({
```

Comment:
I think it could use a distinct name from `message.position` because while it's represented differently in the code vs `messagePosition`, they can be confused when talking about them unless the devs write the names perfectly every time.  Otherwise, "message position" could be interpreted 2 ways.  

I don't mind losing the abbreviation though to `calculatedMessagePosition` or i am open to other names